src/engine/sparqlExpressions/SparqlExpressionPimpl.cpp

```diff
@@ -12,6 +12,13 @@ SparqlExpressionPimpl::SparqlExpressionPimpl(
     std::shared_ptr<SparqlExpression>&& pimpl)
     : _pimpl{std::move(pimpl)} {};
 
+// __________________________________________________________________________
```

Comment:
Can we remove the above constructor for even more "we always have a descriptor" safety?